SlevomatCodingStandard/Sniffs/Attributes/AttributesOrderSniff.php

```diff
@@ -45,8 +50,8 @@ public function process(File $phpcsFile, $attributeOpenerPointer): void
 			return;
 		}
 
-		if ($this->order === []) {
-			throw new UnexpectedValueException('Attributes order not specified.');
+		if ($this->order === [] && $this->orderAlphabetically === false) {
```

Comment:
I think only one of these options should be set so we should throw error even if both are set.